implants/lib/eldritch/src/pivot/mod.rs

```diff
@@ -222,6 +196,6 @@ struct CommandResult {
 
 impl CommandResult {
     fn output(&self) -> Result<String> {
-        Ok(String::from_utf8_lossy(&self.output).try_into()?)
+        Ok(String::from_utf8_lossy(&self.output).to_string())
```

Comment:
Will to_string panic if it fails?

Reply:
It can't fail, clippy provided a lint warning which said this was infallible. UTF8 always supports bytes.